TcUnit-Runner/VisualStudioInstance.cs

```diff
@@ -127,35 +131,61 @@ private void LoadDevelopmentToolsEnvironment(string visualStudioVersion, bool is
             tcAutomationSettings.SilentMode = true; // Only available from TC3.1.4020.0 and above
 
             // Load the correct version of TwinCAT using the remote manager in the automation interface
-            log.Info("Using the TwinCAT remote manager to load TwinCAT version '" + this.tcVersion + "'...");
             ITcRemoteManager remoteManager = dte.GetObject("TcRemoteManager");
-            
-            // If the version is pinned check if the pinned version is available
-            if (isTcVersionPinned)
+
+            string latestTcVersion = "";
+            // Check if Version is installed
+            foreach (var possibleVersion in remoteManager.Versions)
             {
-                log.Info("Project has a pinned version: " + this.tcVersion);
-                foreach (var possibleVersion in remoteManager.Versions)
+                latestTcVersion = Char.ToString(possibleVersion[0]);
```

Comment:
Is there a guarantee that the latest version will always be located at [0]?

Reply:
Actually, this is in the foreach loop... I don't understand this? Does it mean that the latest version will always be the last called...? Is it guaranteed to be the latest version? Maybe it's more suitable to use the Version class here? (see how I use it in method VisualStudioDteAvailable())?

Reply:
> foreach loop...

The foreach loop is to check if optional --TC Version is installed
in the same context i only store the [0] Element of remoteManager.Versions as latestTcVersion

In the default settings TwinCat internally sorts the versions.
The largest version is in [0].
![image](https://user-images.githubusercontent.com/40072981/102311484-436af400-3f6d-11eb-977b-eab6ffa4306a.png)

I can check again if the complier settings have influence on the sorting
![image](https://user-images.githubusercontent.com/40072981/102312356-d8bab800-3f6e-11eb-8421-fac9193b0b3f.png)

I'm not sure right now if I can get the latest version directly from the Version Class? or if I have to compare all versions in a For loop.

Reply:
I see! Yes, my worry is only that Beckhoff will make a change in the future, and that the sorting will be different (the latest will not be available at [0]). You can leave it as it is right now, but please create a new separate ticket/issue here in GitHub for this so that it's not forgotten.